f/radio/f_channelsList.sqf

```diff
@@ -0,0 +1,230 @@
+// F3 - Folk ARPS Long-Range Radio Module
```

Comment:
If we weren't so close to release, I'd suggest moving this to config (`description.ext` et al.). I'll add it as a future idea instead.